docs/core/tools/dotnet-workload-install.md

```diff
@@ -27,6 +27,13 @@ dotnet workload install -?|-h|--help
 
 The `dotnet workload install` command installs one or more *optional workloads*. Optional workloads can be installed on top of the .NET SDK to provide support for various application types, such as [.NET MAUI](/dotnet/maui/what-is-maui) and [Blazor WebAssembly AOT](https://devblogs.microsoft.com/aspnet/asp-net-core-updates-in-net-6-preview-4/#blazor-webassembly-ahead-of-time-aot-compilation).
 
+When the command is in workload-set update mode, workload versions installed:
+
+* Are from the latest workload set or a specified workload-set version.
```

Comment:
```suggestion
* Are from the latest workload set or a workload set version specified in a global.json or --version option.
```

Reply:
Note here that install will not update to a newer workload set if a customer has previously used the --version flag. Not sure how best to document that.

Reply:
I added it as a "for example" statement on the last bullet.